spec/unit/inflecto/inflections/plural_spec.rb

```diff
@@ -24,7 +24,7 @@
     before { object.uncountable(rule) }
 
     it 'removes it from uncountables' do
-      subject.uncountables.should eq([])
+      subject.uncountables.should be_empty
```

Comment:
I personally think the `eq([])` matcher is more exact than `be_empty` I realize this change is from transforming `uncountables` into a `Set` instance. So we IMO should match against `eql(Set.new)`.

Note that I prefer `eql` over `eq` as the former translates to more strict `Object#eql?` where the latter is `Object#==` that does coercion in some cases.